requirements.txt

```diff
@@ -9,4 +9,6 @@ geoalchemy2
 osmnx 
 psycopg2
 imageio
-simplification
\ No newline at end of file
+simplification
+console_progressbar
```

Comment:
Why do we need the `console_progressbar` module? I did not find its usage in the other files. Afaik, this is used for info printing, can we replace it with the more popular `tqdm` module?

Reply:
For now I just removed it but we can think about including tqdm (in a different pull request... this one already got quite confusing)